packages/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts

```diff
@@ -0,0 +1,82 @@
+import { makeStylesCompat, ax } from '@fluentui/react-make-styles';
+import { PresenceBadgeState } from './PresenceBadge.types';
+import { useBadgeStyles } from '../../Badge';
+
+export const usePresenceBadgeRootStyles = makeStylesCompat<PresenceBadgeState>([
+  [
+    null,
+    {
+      padding: 0,
+    },
+  ],
+  [
+    s => s.status === 'busy' || s.status === 'doNotDisturb',
+    theme => ({
+      backgroundColor: theme.alias.color.cranberry.background3,
+      borderColor: theme.alias.color.cranberry.background3,
+    }),
+  ],
+  [
+    s => s.status === 'away',
+    theme => ({
+      backgroundColor: theme.global.palette.yellow.primary,
+      borderColor: theme.global.palette.yellow.primary,
+    }),
+  ],
+  [
+    s => s.status === 'available',
+    theme => ({
+      backgroundColor: theme.global.palette.lightGreen.primary,
+      borderColor: theme.global.palette.lightGreen.primary,
+    }),
+  ],
+  [
+    s => s.status === 'offline',
+    theme => ({
+      backgroundColor: 'transparent',
```

Comment:
I think this one should be neutralBackground1 too.
```suggestion
      backgroundColor: theme.alias.color.neutral.neutralBackground1,
```